### PR TITLE
NurseConnect Change flows

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps please review.  Two things we'll need to look at and talk about, changing of numbers and possibly having alternate states in place of `state_block_active_subs`

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps @gsvr please review

File: src/ussd_nurse.js [hunk 1]
```diff
@@ -65,7 +66,7 @@ go.app = function() {
                 // interrupt = false;
                 // var timeout_opts = opts || {};
                 // timeout_opts.name = name;
-                // return self.states.create('st_timed_out', timeout_opts);
+                // return self.states.create('state_timed_out', timeout_opts);
```
Comment (imsickofmaps) [inline src/ussd_nurse.js:69]: side note: do we have a captured todo for the timeout handling?

Comment (NicolaasDieBaas) [reply src/ussd_nurse.js:69]: we now have --> https://github.com/praekeltfoundation/ndoh-jsbox/issues/42

**Comment by imsickofmaps** [comment]
👍  some simplification can occur when we've excluded inactive identities using the seed-jsbox-utils update that's in progress

**Pull request merged**